website/docs/r/ldap_configuration.html.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: ldap-configuration"
+sidebar_current: "docs-mongodbatlas-resource-ldap-configuration"
+description: |-
+    Provides a LDAP Configuration resource.
+---
+
+# mongodbatlas_ldap_configuration
+
+`mongodbatlas_ldap_configuration` provides an LDAP Configuration resource. This allows ldap configuration to be created.
```

Comment:
Let's include more of the source doc, new - PLEASE make sure to add the correct link where noted:
 `mongodbatlas_ldap_configuration` provides an LDAP Configuration resource. This allows an LDAP configuration for an Atlas project to be crated and managed. This endpoint doesn’t verify connectivity using the provided LDAP over TLS configuration details. To verify a configuration before saving it, use the resource to verify(LINK VERIFY TO website/docs/r/ldap_verify.html.markdown) the LDAP configuration.